starky/src/r1cs2plonk.rs

```diff
@@ -240,20 +261,49 @@ pub fn r1cs2plonk(r1cs: &R1CS<GL>) -> (Vec<PlonkGate>, Vec<PlonkAdd>) {
 }
 
 #[cfg(test)]
-pub mod tests {
-    use crate::compressor12::compressor12_setup::{plonk_setup_render, Options};
-    use crate::r1cs2plonk::r1cs2plonk;
-    //use plonky::bellman_ce::bn256::Bn256;
-    use plonky::field_gl::GL;
+mod test {
+    use super::*;
     use plonky::reader::load_r1cs;
+    use std::fs::File;
+    use std::io::Write;
+    use std::path::Path;
 
+    /// The js dump code as below:
+    /// ```js
+    /// fs.writeFileSync("plonk_constrains_js.json", JSON.stringify(plonkConstraints, (key, value) =>
+    ///     typeof value === 'bigint' ? value.toString() :value
+    /// ));
+    /// fs.writeFileSync("plonk_additions_js.json", JSON.stringify(plonkAdditions, (key, value) =>
+    ///     typeof value === 'bigint' ? value.toString() :value
+    /// ));
+    /// ```
     #[test]
-    #[ignore]
     fn test_r1cs2plonk() {
-        let r1cs = load_r1cs::<GL>("/tmp/circuit.gl.r1cs");
-        let (pc, pa) = r1cs2plonk(&r1cs);
-        println!("pc {}, pa {}", pc.len(), pa.len());
-        let opts = Options { force_bits: 0 };
-        let _plonksetupinfo = plonk_setup_render(&r1cs, &opts, "/tmp/c12.pil");
+        let CIRCUIT: String = std::env::var("CIRCUIT")
```

Comment:
We have better not use env variable for the test, it makes UT too complicated.  Just generate a tmp file, or upload a pil in data directory, then test on it.